erpnext/subcontracting/doctype/subcontracting_inward_order/test_subcontracting_inward_order.py

```
@@ -51,6 +51,49 @@ def test_customer_provided_item_cost_field(self):
 		for item in rm_in.get("items"):
 			self.assertEqual(item.customer_provided_item_cost, 15)
 
+	def test_customer_provided_item_cost_with_multiple_receipts(self):
+		"""
+		Validate that rate is calculated correctly (Weighted Average) when multiple receipts
+		occur for the same SCIO Received Item.
+		"""
+		so, scio = create_so_scio()
```

Ruff (0.14.13)

Unpacked variable `so` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)